cube/src/main/java/zone/cogni/asquare/cube/convertor/data2shacl/Configuration.java

```diff
@@ -17,6 +18,8 @@ public class Configuration {
 
   private Map<Set<String>, String> typeTranslations = new HashMap<>();
 
+  private Set<String> prioritisedNamespaces = new HashSet<>();
```

Comment:
@michielrogissart-cognizone how would set make it work? order is important I would say. first take properties of first namespace, then second etc.